### PR TITLE
feat(locksmith) - keysOperations

### PR DESCRIPTION
**Pull request by kalidiagne**
# Description
this PR is part of  https://github.com/unlock-protocol/unlock/issues/9213, adds `keysOperations`, with the operations needed to merge keys information with metadata

# Issues

Fixes #
Refs #

# Checklist:

- [x] 1 PR, 1 purpose: my Pull Request applies to a single purpose
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the docs to reflect my changes if applicable
- [x] I have added tests (and stories for frontend components) that prove my fix is effective or that my feature works
- [x] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

File: locksmith/src/operations/keysOperations.ts [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+}) {
+  const web3Service = new Web3Service(networks)
+  const isLockOwner = await web3Service.isLockManager(
+    lockAddress,
+    loggedInUserAddress,
+    network
+  )
+
+  let metadataItems = []
+  const client = new keysByQuery(network)
```
Comment (julien51) [inline locksmith/src/operations/keysOperations.ts:101]: Let's use `graphQLClient`

**Review by julien51 — APPROVED** [review]
Small nits and then LG!

**Comment by julien51** [comment]
Lowering the coverage is not right. Please identify where you are decreasing coverage and add the missing tests.

**Comment by kalidiagne** [comment]
Added more tests and restored test coverage as default

**Review by julien51 — APPROVED** [review]

**Pull request merged**